src/main/java/com/faforever/api/security/FafUserDetailsService.java

```diff
@@ -23,6 +23,9 @@ public FafUserDetailsService(UserRepository userRepository) {
 
   @Override
   public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+    if (userRepository.findOneByLoginIgnoreCase(username) == null) {
+      throw new UsernameNotFoundException("Username can't be null");
```

Comment:
This error message is misleading; it's not that the username is null, but that the username could not be found.
`"User could not be found: " + username` would be a proper message.